hive/src/test/java/org/apache/iceberg/hive/TestHiveMetastore.java

```diff
@@ -111,6 +111,7 @@ private HiveConf newHiveConf(int port) {
     HiveConf newHiveConf = new HiveConf(new Configuration(), TestHiveMetastore.class);
     newHiveConf.set(HiveConf.ConfVars.METASTOREURIS.varname, "thrift://localhost:" + port);
     newHiveConf.set(HiveConf.ConfVars.METASTOREWAREHOUSE.varname, "file:" + hiveLocalDir.getAbsolutePath());
+    newHiveConf.set(HiveConf.ConfVars.METASTORE_TRY_DIRECT_SQL.varname, "false");
```

Comment:
What does this do?

Reply:
This is used to eliminate warning in the unit test.

Reply:
What warning?

Reply:
WARN org.apache.hadoop.hive.metastore.ObjectStore - Direct SQL failed, falling back to ORM
java.lang.ClassCastException: org.apache.derby.impl.jdbc.EmbedClob cannot be cast to java.lang.String.